### PR TITLE
update Connectable to include sthis

### PR DESCRIPTION
**Pull request by mschoch**
and export it from fireproof

### CODE REVIEW

**Comment by mschoch** [comment]
@jchris this was what I tried to make Connectable usable again in the connect repo.

**Comment by jchris** [comment]
thanks I will preview it when i get into that layer which is soon

**Comment by mschoch** [comment]
I realize now this is wrong.  We export the blockstore package under the name 'bs', but I didn't realize that until now.

File: src/blockstore/connection-base.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface Connectable {
     readonly ebOpts: BlockstoreRuntime;
   };
   readonly name?: string;
+  readonly sthis: SuperThis;
```
Comment (jchris) [inline src/blockstore/connection-base.ts:17]: what about this?

**Pull request closed without merging**